### PR TITLE
bumping ev version

### PR DESCRIPTION
**Pull request by caueguerra**
This is needed for cool.io to be compiled under the new OS X version, El Capitan

### CODE REVIEW

File: ext/libev/ev.c [hunk 1]
```diff
@@ -3335,15 +3402,19 @@ time_update (EV_P_ ev_tstamp max_block)
...
-VALUE ev_backend_poll(void **args)
+VALUE ev_backend_poll(void *ptr)
 {
-  struct ev_loop *loop = (struct ev_loop *)args[0];
-  ev_tstamp waittime = *(ev_tstamp *)args[1];
-  backend_poll (EV_A_ waittime);
-  return Qnil;
+  struct ev_poll_args *args = (struct ev_poll_args *)ptr;
+  struct ev_loop *loop = args->loop;
+  backend_poll (EV_A_ args->waittime);
```
Comment (repeatedly) [inline ext/libev/ev.c]: `return Qnil;` is needed to avoid warning.

File: ext/libev/ev.c [hunk 2]
```diff
@@ -3335,15 +3402,19 @@ time_update (EV_P_ ev_tstamp max_block)
     }
 }
 
-/* ########## COOLIO PATCHERY HO! ########## */
+/* ########## NIO4R PATCHERY HO! ########## */
```
Comment (repeatedly) [inline ext/libev/ev.c]: This patch is copied from nio4r?

File: ext/libev/ev.c [hunk 3]
```diff
@@ -3351,9 +3422,9 @@ VALUE ev_backend_poll(void **args)
 int
 ev_run (EV_P_ int flags)
 {
-/* ########## COOLIO PATCHERY HO! ########## */
+/* ########## NIO4R PATCHERY HO! ########## */
```
Comment (repeatedly) [inline ext/libev/ev.c]: ditto

**Comment by caueguerra** [comment]
@repeatedly fixed

**Comment by repeatedly** [comment]
Thanks!

**Comment by repeatedly** [comment]
Released v1.4.0.

**Pull request merged**